components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/test/java/org/wso2/carbon/identity/application/authentication/framework/session/extender/processor/SessionExtenderProcessorTest.java

```diff
@@ -71,29 +59,4 @@ public void testCanHandle() {
         assertTrue(sessionExtenderProcessor.canHandle(sessionExtenderRequest), "Cannot handle valid " +
                 "SessionExtenderRequest.");
     }
-
-    @Test(expectedExceptions = NullPointerException.class)
-    public void testProcessWithSessionKey() throws Exception {
```

Comment:
Removed this test case considering that its just expecting a NPE, which should not happen in the runtime.